rdrf/rdrf/frontend/src/pages/proms_page/components/question.tsx

```diff
@@ -201,12 +204,15 @@ class Question extends React.Component<QuestionInterface, object> {
         );
     }
 
-    public renderInput(question: any) {
+    public renderInput(question: any, type: InputType) {
         let defaultValue = ""
         if (this.props.answers[question.cde]) {
             defaultValue = this.props.answers[question.cde]
         }
-
+        let pattern = ".*";
+        if (type === "number"){
+            pattern = "-?[0-9]*";
```

Comment:
This should be "-?[0-9]+"    as a single "-" or empty string will be accepted otherwise.